index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tasnimul Hasan Portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+
+</head>
+<body>
+
+    <header>
+        <div class="nav-menu-content">
+
+            <div class="logo"><a class=mylogo> My Logo </a></div>
+
+            <nav>
+                <ul class="nav-menu">
+                    <li class="nav-items"><a href="#">Portfolio</a></li>
+                    <li class="nav-items"><a href="#">About</a></li>
+                    <li class="nav-items"><a href="#">Contact</a></li>
+                  </ul>
+
+
+                    <div class="hamburger">
+
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                    </div>
+            </nav>
+        </div>
+
+    </header>
+    <br>
+    <br>
+    <section class="homepage">
+        <div class="introduction">
+            <div class="intro"><h1>I'm Tasnimul <br>Glad to meet you!</h1></div>
+            <br>
+            <br>
+            <br>
+            <div class="me">
+                <p>
+                    I am a software developer. I have done some interesting projects using
+                    react js, node js and ruby on rails. I am learning new things and
+                    implementing them instantly. I can help you build a project, feature
+                    or website look through some of work and experience if you like what
+                    you see and have a project you need coded, don't hesitate to contact
+                    me.
+                </p>
+            </div>
+            <br>
+            <div class="connect">
+                <h3><a> Let's connect </a> </h3>
+            </div>
+            <br>
+            <div class="social">
+                <ul>
+                    <li class="icon"><a href=""><img src="Image/twitter.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/github.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/medium.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/angellist.png" alt="" /></a></li>
```

Comment:
## Optional

Figma Design          |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/69638013/153697940-b2e12c4b-61d8-447e-b875-1d560ecf0cc2.png)   |  ![smicons](https://user-images.githubusercontent.com/69638013/153698039-b57862a1-0948-479e-9a22-1d4835f9f596.PNG)

- Kindly add the missing icon, so your design looks like the figma design, as this was missed by the previous reviewer 👍🏿

Reply:
linked in image has been added to page